<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>project</class>
 <widget class="QWidget" name="project">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>750</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>github</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_2">
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_2">
     <property name="topMargin">
      <number>0</number>
     </property>
     <item>
      <widget class="QLabel" name="repoNameLabel">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Expanding" vsizetype="Preferred">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="font">
        <font>
         <pointsize>18</pointsize>
         <bold>true</bold>
        </font>
       </property>
       <property name="text">
        <string>repo</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="infoLabel">
       <property name="styleSheet">
        <string notr="true">color: rgb(255, 0, 0);</string>
       </property>
       <property name="text">
        <string/>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QVBoxLayout" name="verticalLayout">
     <item>
      <layout class="QVBoxLayout" name="verticalLayout_4">
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout">
         <item>
          <layout class="QFormLayout" name="formLayout_2">
           <property name="fieldGrowthPolicy">
            <enum>QFormLayout::AllNonFixedFieldsGrow</enum>
           </property>
           <item row="0" column="0">
            <widget class="QLabel" name="nombreCompletoLabel">
             <property name="styleSheet">
              <string notr="true">font: 700 9pt &quot;Segoe UI&quot;;</string>
             </property>
             <property name="text">
              <string>Nombre completo</string>
             </property>
            </widget>
           </item>
           <item row="0" column="1">
            <widget class="QLineEdit" name="nombreCompletoLineEdit">
             <property name="readOnly">
              <bool>true</bool>
             </property>
            </widget>
           </item>
           <item row="2" column="0">
            <widget class="QLabel" name="descrLabel">
             <property name="styleSheet">
              <string notr="true">font: 700 9pt &quot;Segoe UI&quot;;</string>
             </property>
             <property name="text">
              <string>Descripción</string>
             </property>
            </widget>
           </item>
           <item row="2" column="1">
            <widget class="QLineEdit" name="descLineEdit">
             <property name="readOnly">
              <bool>true</bool>
             </property>
            </widget>
           </item>
           <item row="3" column="0">
            <widget class="QLabel" name="fechaCreaLabel">
             <property name="styleSheet">
              <string notr="true">font: 700 9pt &quot;Segoe UI&quot;;</string>
             </property>
             <property name="text">
              <string>Fecha creación</string>
             </property>
            </widget>
           </item>
           <item row="3" column="1">
            <widget class="QLineEdit" name="fechaCreaLineEdit">
             <property name="readOnly">
              <bool>true</bool>
             </property>
            </widget>
           </item>
           <item row="4" column="0">
            <widget class="QLabel" name="ultimaActualizacionLabel">
             <property name="styleSheet">
              <string notr="true">font: 700 9pt &quot;Segoe UI&quot;;</string>
             </property>
             <property name="text">
              <string>Última actualización</string>
             </property>
            </widget>
           </item>
           <item row="4" column="1">
            <widget class="QLineEdit" name="ultimaActualizacionLineEdit">
             <property name="readOnly">
              <bool>true</bool>
             </property>
            </widget>
           </item>
           <item row="5" column="0">
            <widget class="QLabel" name="pgIniLabel">
             <property name="styleSheet">
              <string notr="true">font: 700 9pt &quot;Segoe UI&quot;;</string>
             </property>
             <property name="text">
              <string>Página de inicio</string>
             </property>
            </widget>
           </item>
           <item row="5" column="1">
            <widget class="QLineEdit" name="pgIniLineEdit">
             <property name="readOnly">
              <bool>true</bool>
             </property>
            </widget>
           </item>
           <item row="6" column="0">
            <widget class="QLabel" name="lenguajeLabel">
             <property name="styleSheet">
              <string notr="true">font: 700 9pt &quot;Segoe UI&quot;;</string>
             </property>
             <property name="text">
              <string>Lenguaje</string>
             </property>
            </widget>
           </item>
           <item row="6" column="1">
            <widget class="QLineEdit" name="lenguajeLineEdit">
             <property name="readOnly">
              <bool>true</bool>
             </property>
            </widget>
           </item>
           <item row="7" column="0">
            <widget class="QLabel" name="forksLabel">
             <property name="styleSheet">
              <string notr="true">font: 700 9pt &quot;Segoe UI&quot;;</string>
             </property>
             <property name="text">
              <string>Forks</string>
             </property>
            </widget>
           </item>
           <item row="7" column="1">
            <widget class="QLineEdit" name="forksLineEdit">
             <property name="readOnly">
              <bool>true</bool>
             </property>
            </widget>
           </item>
           <item row="8" column="0">
            <widget class="QLabel" name="estrellasLabel">
             <property name="styleSheet">
              <string notr="true">font: 700 9pt &quot;Segoe UI&quot;;</string>
             </property>
             <property name="text">
              <string>Estrellas</string>
             </property>
            </widget>
           </item>
           <item row="8" column="1">
            <widget class="QLineEdit" name="estrellasLineEdit">
             <property name="readOnly">
              <bool>true</bool>
             </property>
            </widget>
           </item>
           <item row="9" column="0">
            <widget class="QLabel" name="licenciaLabel">
             <property name="styleSheet">
              <string notr="true">font: 700 9pt &quot;Segoe UI&quot;;</string>
             </property>
             <property name="text">
              <string>Licencia</string>
             </property>
            </widget>
           </item>
           <item row="9" column="1">
            <widget class="QLineEdit" name="licenciaLineEdit">
             <property name="readOnly">
              <bool>true</bool>
             </property>
            </widget>
           </item>
           <item row="10" column="0">
            <widget class="QLabel" name="commitsLabel">
             <property name="styleSheet">
              <string notr="true">font: 700 9pt &quot;Segoe UI&quot;;</string>
             </property>
             <property name="text">
              <string>Commits</string>
             </property>
            </widget>
           </item>
           <item row="10" column="1">
            <widget class="QWidget" name="commitsWidget" native="true">
             <property name="sizePolicy">
              <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
               <horstretch>0</horstretch>
               <verstretch>0</verstretch>
              </sizepolicy>
             </property>
            </widget>
           </item>
           <item row="1" column="0">
            <widget class="QLabel" name="creadorLabel">
             <property name="styleSheet">
              <string notr="true">font: 700 9pt &quot;Segoe UI&quot;;</string>
             </property>
             <property name="text">
              <string>Creador</string>
             </property>
            </widget>
           </item>
           <item row="1" column="1">
            <widget class="QLineEdit" name="creadorLineEdit">
             <property name="readOnly">
              <bool>true</bool>
             </property>
            </widget>
           </item>
          </layout>
         </item>
        </layout>
       </item>
      </layout>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_3">
     <property name="topMargin">
      <number>0</number>
     </property>
     <item>
      <widget class="QPushButton" name="actualizarInformacionBtn">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="text">
        <string>Actualizar información</string>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="horizontalSpacer">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QPushButton" name="abrirCarpetaBtn">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="text">
        <string>Abrir carpeta</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
  <action name="actionConfiguraci_n">
   <property name="text">
    <string>Configuración</string>
   </property>
  </action>
  <action name="actionAcerca_de">
   <property name="text">
    <string>Acerca de</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections/>
</ui>
